edge-modules/api-proxy-module/azure-iot-mqtt/README.md

```diff
@@ -0,0 +1,34 @@
+An Azure IoT client library. It implements [the Azure IoT Hub MQTT protocol.](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support)
+
+
+# Features
+
+- Device client
+	- Receive initial twin state and updates
+	- Receive and respond to direct method requests
+
+- Module client
+	- Receive and respond to direct method requests
+
+- Supports MQTT and MQTT-over-WebSocket protocols.
+
+- Transparently reconnects when connection is broken or protocol errors, with back-off.
+
+- Standard futures 0.1 and tokio 0.1 interface. The client is just a `futures::Stream` of events received from the server.
```

Comment:
is this too specific here?

Reply:
This md file is actually the SDK from Arnav. This should not be part of the review.